pkg/master/master.go

```diff
@@ -96,6 +96,12 @@ const (
 	DefaultEtcdPathPrefix = "/registry"
 )
 
+var (
+	// allAPIVersions records all AVIVersions that are supported at /apis, in the
+	// form of "group/version"
+	allAPIVersions []string
```

Comment:
Isn't the third party resource part of Master?  Maybe I'm confused but the
scope should be the master object.

On Tue, Sep 15, 2015 at 12:49 PM, Chao Xu <notifications@github.com> wrote:

> In pkg/master/master.go
> <https://github.com/kubernetes/kubernetes/pull/13955#discussion_r39534611>
> :
>
> > @@ -96,6 +96,12 @@ const (
> >  	DefaultEtcdPathPrefix = "/registry"
> >  )
> >
> > +var (
> > +	// allAPIVersions records all AVIVersions that are supported at /apis, in the
> > +	// form of "group/version"
> > +	allAPIVersions []string
>
> Ah, I forgot to leave a note here. It's package level because we will need
> to add/delete elements in this variable when installing/deleting thirdparty
> resource,
> https://github.com/kubernetes/kubernetes/blob/master/pkg/master/master.go#L777
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/13955/files#r39534611>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift